src/app/lorax/directives/share.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @fileOverview Share directive
+ * @author <a href="mailto:chris@work.co">Chris James</a>
+ * @author <a href="mailto:sneethling@mozilla.com">Schalk Neethling</a>
+ */
+define(['angular', 'jquery'], function (angular, $) {
+    'use strict';
+
+    /**
+     * Share directive
+     */
+    var ShareDirective = function () {
+        return {
+            restrict: 'A',
+            scope: true,
+            controller: ShareController
+        };
+    };
+
+    /**
+    * Controller for Share directive
+    * @constructor
+    */
+    var ShareController = function (
+        $scope,
+        dataService,
+        routesService
+    ) {
+
+        console.log('ShareController');
+
+        this._$scope = $scope;
+        this._dataService = dataService;
+        this._routesService = routesService;
+
+        // listen for $broadcast of 'share' which will be broadcast from:
+        // /src/app/lorax/controller/core.js
+        $scope.$on('share', this.share.bind(this));
+
+        this._dataService.getMain().then(function (model) {
+            this._$scope.content = {
+                share: model.getServices().share
```

Comment:
Actually, there should be no occurrences of that. It is triggered by
https://github.com/mozilla/lorax/pull/25/files#diff-e249faefed5757034596c5096d33dab6R51

And the broadcast happens in:
https://github.com/mozilla/lorax/pull/25/files#diff-d417c7d505ad93462e1f4299e0b7ea7eR73

But the thing is, it has not even gotten to the click event yet, the multiple init is happening on loading the main landing page i.e. something like 127.0.0.1:9000